dev/com.ibm.ws.channelfw/src/com/ibm/ws/udpchannel/internal/UDPChannel.java

```diff
@@ -314,12 +315,13 @@ protected UDPChannelConfiguration getConfig() {
     /**
      * Verify whether the remote address is allowed to communicated with the
      * channel.
+     *
      * @param remoteAddr
      * @return boolean - false means it is denied
      */
     public boolean verifySender(InetAddress remoteAddr) {
         boolean returnValue = true;
-
+        /* 18752 */
```

Comment:
what is this comment needed for?

Reply:
Apologies this was a marker for places to look for to make common - I thought that I had removed them all - apologies. Gone.